combat-trainer.lic

```diff
@@ -1127,8 +1127,15 @@ class SpellProcess
 
   def check_health(game_state)
     return if game_state.casting
-    return unless DRStats.empath?
-    return if health > 95 && @wounds.empty?
+    return unless DRStats.empath? || DRStats.necromancer?
+    return if health > 95 && @wounds.empty? && DRStats.empath?
+    return if health > 90 && DRStats.necromancer?
+
+    if DRStats.necromancer? && @necromancer_healing['Siphon Vitality']
+      @necromancer_healing['Siphon Vitality']['prep'] = 'target'
+      prepare_spell(@necromancer_healing['Siphon Vitality'], game_state, true)
```

Comment:
I dont think true/false should be a parameter to prepare_spell. The intention should be captured well enough by modifying the spell data or game_state, e.g. line 1135

Reply:
Not sure what the issue is. prepare_spell uses cambrinth or attunement based on a setting and then based on the mind state of said skill, this is a forced cambrinth use for that. I am seriously so confused as to why this is an issue? It makes perfect sense, and defaults to false if not. There is actually no requirement for it to be put through, but if put through, it forces it. I would need a whole other setting for this. That is, unless we change all spells to be able to force cambrinth use regardless of the mind state of attunement vs arcana.